service/validator.go

```diff
@@ -54,7 +55,15 @@ func NewValidator(audienceConfig config.AudienceConfig, opts ...ValidatorOption)
 	}
 
 	if len(serviceValidator.issuer) == 0 {
-		serviceValidator.issuer = serviceValidator.realmURL()
+		var err error
+		serviceValidator.issuer, err = serviceValidator.issuerURL()
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	if serviceValidator.realm != "addons" {
+		serviceValidator.baseURL += "/auth/realms"
 	}
```

Comment:
can you put an additional comment here explaining this? E.g `Keycloak requires this path prefix internally on auth.services.bitrise.io domain (for addons this service is available on app.bitrise.io)`?